<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecommendationsWidget</class>
 <widget class="QWidget" name="RecommendationsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_containment">
     <item>
      <widget class="QLabel" name="label_containment">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Сдерживание</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea_containment">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_containment">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>306</width>
          <height>452</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTextEdit" name="textEdit_cont"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_fixes">
     <item>
      <widget class="QLabel" name="label_fixes">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Устранение</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea_fixes">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_fixes">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>306</width>
          <height>452</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTextEdit" name="textEdit_fixes"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_restore">
     <item>
      <widget class="QLabel" name="label_restore">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Восстановление</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea_restore">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_restore">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>306</width>
          <height>452</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTextEdit" name="textEdit_restore"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
